### PR TITLE
USWDS - POAM: November ‘24

### PR DESCRIPTION
**Pull request by cathybaptista**
# Summary

Updated vulnerable dependencies, updated indirect dependency for elliptic
## Breaking change

This is not a breaking change.

## Related issue

uswds/uswds-team#413

## Preview link

[Preview link:](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cb-poam-nov-24/?path=/story/design-tokens-fonts--fonts)

## Major changes

Before:

```node
57 vulnerabilities (28 moderate, 29 high)
```

After:

```node
57 vulnerabilities (28 moderate, 29 high)
```

Node.js updated from `v 20.18.1 to 22.11.0` to get the latest LTS version
`(v22.11.0   (Latest LTS: Jod))
`
## Testing and review

1. Pull down and run `npm install`
2. Run `node -v` and check major version is 22 
3. Confirm no installation errors
4. Run Compile scripts
5. Confirm no build errors
6. Run `npm run test`
7. Confirm no testing or linting errors 
9. Start local or visit preview link
10. Confirm there are no visual regressions

## Dependency updates

| **Dependency name** | **Old version** | **New version** |
| --- | --- | --- |
| @babel-core | 7.25.7 | 7.26.0 |
| @babel/preset-env  | 7.25.7 | 7.26.0 |
| axe-core | 4.10.0 | 4.10.2
| cross-spawn | 7.0.3 | 7.0.6
| html-webpack-plugin | 5.6.0 | 5.6.3
| mocha | 10.7.3 | 10.8.2
| postcss | 8.4.47 | 8.4.49
| sass | 1.79.4 | 1.81.0
| sass-embedded | 1.79.4 | 1.81.0
| snyk | 1.1293.1 | 1.1294.1
| stylelint | 16.9.0 | 16.10.0
| typescript | 5.6.2 | 5.7.2
| webpack | 5.95.0 | 5.96.1

### CODE REVIEW

**Comment by amyleadem** [comment]
Comment from @mejiaj in #6211: 

> Can we update node in #6208 @mahoneycm @cathybaptista?

Are you able to update node to [LTS](https://nodejs.org/en) in this PR? LTS is currently v22, but our [.nvmrc file](https://github.com/uswds/uswds/blob/develop/.nvmrc) says v20.

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-20
+lts/*
```
Comment (mejiaj) [inline .nvmrc:1]: > [!NOTE]
> Installs v22.11.0 and npm 10.9.0

**Review by mejiaj — DISMISSED** [review]
LGTM, thanks @cathybaptista.

I tested the following:
- [x] Compared vulns in develop to this branch [`56 vulnerabilities (29 moderate, 27 high)`]
- [x] Confirmed there aren't regressions to JS linting, building twig, and running storybookJS.

---

There are a few more updates we can make, but these don't have an impact on vulns:

| Package       | Current version      | Available update      |
| :------------- | :-------- | :-------- |
| cross-spawn   | 7.0.5    | 7.0.6    |
| sass          | 1.80.7   | 1.81.0   |
| sass-embedded | 1.80.7   | 1.81.0   |
| snyk          | 1.1294.0 | 1.1294.1 |
| typescript    | 5.6.3    | 5.7.2    |

**Review by amyleadem — APPROVED** [review]
LGTM!
Tested with Node v22.11.0. 
Vulnerability count: `55 vulnerabilities (29 moderate, 26 high)`
- [x] Confirm we can run `npm install`, `npm start`, `npm build:html` and `npm run test` without error. 
- [x] Confirmed Storybook components render as expected
- [x] Confirm dependency table matches the changes in package.json

**Review by mejiaj — APPROVED** [review]

**Pull request merged**